### ISSUE TITLE
Passing the redis database number with launch parameters

### ISSUE THREAD

**Issue by MinoMino** [MinoMino/minqlx#3]
From IRC:

```
<mattiZed> but being able to pass the database to use via commandline would be nice, if easy to implement
<mattiZed> i guess most people will spawn their servers via supervisor and launch script
<@Mino> Yeah
<@Mino> I'll add a cvar to qlds
<@Mino> then you can do like a +set sv_redis 1
```

**Comment by mattiZed** [comment]
This might somehow coincide with issue #7 i just created :)

**Comment by MinoMino** [comment]
Addressed in v0.0.2.
